auslib/db.py

```diff
@@ -1783,21 +1783,18 @@ def getPotentialRequiredSignoffs(self, affected_rows, transaction=None):
         # to them. We need to find these Rules, and then return _their_
         # Required Signoffs.
         if info:
-            # get all rules as one query
-            q_rules = [self.db.rules.rule_id.in_(tuple([rule_id for row in info for rule_id in row["rule_ids"]]))]
-            all_rules = self.db.rules.select(where=q_rules, transaction=transaction)
-
-            # map rules according to rule_id
-            rules_map = {rule["rule_id"]: rule for rule in all_rules}
+            relevant_rules = []
```

Comment:
Nit: This chunk can be simplified by a list-comprehension.

```suggestion
            relevant_rules = [rule_info for row in info for rule_info in row["rule_info"].values()]
```